app/models/conference.rb

```diff
@@ -4,7 +4,9 @@ class Conference < ActiveRecord::Base
 
   has_many :attendances
   has_many :attendees, through: :attendances, source: :user
+
   validates :name, :round, presence: true
+  validate :chronology # custom validator
```

Comment:
I would go for a more "speaking name". How do you like ` :must_end_after_it_started` or something like this?